.github/workflows/build-and-test.yml

```diff
@@ -30,7 +26,21 @@ jobs:
 
       - name: Build all on ${{matrix.os}}
         run: make all
-        
+
+
+      - name: Install Snyk CLI
+        run: npm install -g snyk
```

Comment:
Better to always have a locked/known version

Reply:
Made changes